### ISSUE TITLE
Chart keyboard navigation not working with Chart.js 3

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#1559]
If `chartjs_3` is set to true, the tooltip keyboard navigation doesn't work. It's supposed to allow users to cycle through the tooltips using the arrow keys.
